apps/aecore/src/aec_pow_cuckoo.erl

```diff
@@ -20,8 +20,13 @@
 -compile([export_all, nowarn_export_all]).
 -include_lib("eunit/include/eunit.hrl").
 -endif.
+-include("pow.hrl").
 
--define(DEFAULT_CUCKOO_ENV, {"mean30", "-t 5", 30}). %% -m 7 caused crash
+-define(DEFAULT_CUCKOO_ENV, {"mean30", "-t 1", 30}). %% -m 7 caused crash
```

Comment:
Is this change intentional? There are issues with running cuckoo miner multi-threaded but I thought we concluded that for the actual miner (running in a separate thread) it will be better to use (5) threads... In some of our tests we'd like to run it single-threaded though!

Reply:
@akorosmezey If this is for making tests stable, I believe it is not needed thanks to https://github.com/aeternity/epoch/commit/aba0fda1f7dec68bdd74a8776c9286dbf1662e44 (included in master hence in this PR). Please elaborate.

Reply:
Thanks for spotting, Fixed in next commit 6a49d96